src/lib/dag.js

```diff
@@ -19,62 +36,69 @@ export function explainDagPb (node) {
     format = `unixfs`
   } catch (err) {
     // dag-pb but not a unixfs.
-    console.log(err)
+    // console.log(err)
   }
+
   return {
+    cid,
     type: 'dag-pb',
-    format,
-    ...node
+    data: node.data,
+    links: normaliseDagPbLinks(node),
+    size: node.size,
+    format
   }
 }
 
-export function explainDagCbor (obj) {
-  const links = findIpldLinks(obj)
+/*
+ * Convert DagLink shape into normalized form that can be used interchangeably
+ * with links found in dag-cbor
+ */
+export function normaliseDagPbLinks (node) {
+  return node.links.map(({name, size, multihash}) => ({
+    path: name,
```

Comment:
:bike: :house:   since `path` is actually either a `path` (cbor) or a `name` (pb), perhaps `context` or `label` would be a less confusing term?